index.js

```diff
@@ -214,6 +214,14 @@ function showSearch() {
     });
 }
 
+/*
+ * Opens the bookmark menu
+ */
+function showBookmarks() {
+    let activeWindow = windowUtils.getMostRecentBrowserWindow();
+    UITour.showMenu(activeWindow, 'bookmarks');
```

Comment:
Yep, I'll throw it in a try catch. It is the very similar concept to highlighting, so this will work with a few line deletions.